### PR TITLE
AI PL Launch - Update code.org/ai page

### PR DESCRIPTION
**Pull request by kelbyhawn**
Update https://code.org/ai to include AI PL offerings:
- Add How AI Works curricula card
- Add AI 101 for Teachers PL card
- Add two new AI videos 
  - Also added to https://code.org/educate/resources/videos
- Did a tiny bit of refactoring to update the `action-block` elements w/ newer patterns

**Related PR, dependant on this one:** 
- https://github.com/code-dot-org/code-dot-org/pull/53146

**Jira ticket:** https://codedotorg.atlassian.net/browse/ACQ-752

----

## Before 
Screenshot 2023-08-02 at 1 13 24 PM

--

Screenshot 2023-08-02 at 1 13 30 PM

--

(The rest of the video order remains the same)

Screenshot 2023-08-02 at 1 13 36 PM

## After
Screenshot 2023-08-07 at 9 39 25 AM

--

Screenshot 2023-08-02 at 1 14 47 PM

--

(The rest of the video order remains the same)

Screenshot 2023-08-02 at 1 14 54 PM

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb [hunk 1]
```diff
@@ -32,6 +32,18 @@ We are building a growing library of educational videos available for re-use by
     <%=view :display_video_thumbnail, id: "introducing_ai", video_code: "Ok-xpKjKp2g", caption: "Introducing How AI Works", play_button: 'center', letterbox: 'false', facebook: facebook, twitter: twitter %>
   </div>
 
+  <div class="col-50" style="float: left; padding: 10px; margin-bottom: 40px">
```
Comment (megcrenshaw) [inline pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb:35]: Learning a lot from these column styles!

**Review by megcrenshaw — APPROVED** [review]
🥳 🙌

File: pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb (hunk 1 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb:35]: Yeah! The `col-50` class is part of the `brstrap.css` stylesheet. (This is a legacy page though so the padding and margin aren't ideal.)

File: pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb [hunk 2]
```diff
@@ -32,6 +32,18 @@ We are building a growing library of educational videos available for re-use by
...
+    <% facebook = {:u=>'https://youtu.be/T45kEEBzrT8'}
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb]: Update cobranded `How Chatbots and Large Language Models Work` video here

File: pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb [hunk 3]
```diff
@@ -32,6 +32,18 @@ We are building a growing library of educational videos available for re-use by
...
+    twitter = {:url=>'https://youtu.be/T45kEEBzrT8', :related=>'codeorg', :text=>'How Chatbots and Large Language Models Work @codeorg'} %>
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb]: Update cobranded `How Chatbots and Large Language Models Work` video here

File: pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb [hunk 4]
```diff
@@ -32,6 +32,18 @@ We are building a growing library of educational videos available for re-use by
...
+    <%=view :display_video_thumbnail, id: "ai_chatbots", video_code: "T45kEEBzrT8", caption: "How Chatbots and Large Language Models Work", play_button: 'center', letterbox: 'false', download_path: "//videos.code.org/ai_series/ai-chatbots.mp4", facebook: facebook, twitter: twitter %>
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb]: Update cobranded `How Chatbots and Large Language Models Work` video here, and new AWS download link

File: pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb [hunk 5]
```diff
@@ -32,6 +32,18 @@ We are building a growing library of educational videos available for re-use by
...
+  </div>
+
+  <div class="col-50" style="float: left; padding: 10px; margin-bottom: 40px">
+    <% facebook = {:u=>'https://youtu.be/gD1bEC9q54k'}
+    twitter = {:url=>'https://youtu.be/gD1bEC9q54k', :related=>'codeorg', :text=>'Does AI Have Creativity and Imagination? @codeorg'} %>
+    <%=view :display_video_thumbnail, id: "ai_creativity", video_code: "gD1bEC9q54k", caption: "Does AI Have Creativity and Imagination?", play_button: 'center', letterbox: 'false', download_path: "//videos.code.org/ai_series/ai-creativity.mp4", facebook: facebook, twitter: twitter %>
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb]: Update `Does AI Have Creativity and Imagination?` cobranded video here, and new AWS download link

File: pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb [hunk 6]
```diff
@@ -32,6 +32,18 @@ We are building a growing library of educational videos available for re-use by
...
   </div>
 
+  <div class="col-50" style="float: left; padding: 10px; margin-bottom: 40px">
+    <% facebook = {:u=>'https://youtu.be/T45kEEBzrT8'}
+    twitter = {:url=>'https://youtu.be/T45kEEBzrT8', :related=>'codeorg', :text=>'How Chatbots and Large Language Models Work @codeorg'} %>
+    <%=view :display_video_thumbnail, id: "ai_chatbots", video_code: "T45kEEBzrT8", caption: "How Chatbots and Large Language Models Work", play_button: 'center', letterbox: 'false', download_path: "//videos.code.org/ai_series/ai-chatbots.mp4", facebook: facebook, twitter: twitter %>
+  </div>
+
+  <div class="col-50" style="float: left; padding: 10px; margin-bottom: 40px">
+    <% facebook = {:u=>'https://youtu.be/gD1bEC9q54k'}
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb]: Update `Does AI Have Creativity and Imagination?` cobranded video here

File: pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb [hunk 7]
```diff
@@ -32,6 +32,18 @@ We are building a growing library of educational videos available for re-use by
...
+    twitter = {:url=>'https://youtu.be/gD1bEC9q54k', :related=>'codeorg', :text=>'Does AI Have Creativity and Imagination? @codeorg'} %>
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb]: Update `Does AI Have Creativity and Imagination?` cobranded video here

**Comment by kelbyhawn** [comment]
@dmcavoy I added comments to the lines that need video replacements and on [this commit](https://github.com/code-dot-org/code-dot-org/pull/53136/commits/78fbe0bbef6e0fced34839ab81e1897c4f56e88b) for the partials.

File: pegasus/sites.v3/code.org/views/ai_vid_chatbots.erb [hunk 8]
```diff
@@ -1,3 +1,4 @@
 <div class="col-50" style="float: left; padding: 10px; margin-bottom: 40px">
+  <!-- Update cobranded video here, and new AWS download link -->
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/views/ai_vid_chatbots.erb:2]: Update cobranded video here, and new AWS download link

File: pegasus/sites.v3/code.org/views/ai_vid_chatbots.erb [hunk 9]
```diff
@@ -1,3 +1,4 @@
...
   <%=view :display_video_thumbnail, id: "ai_chatbots", video_code: "T45kEEBzrT8", caption: hoc_s(:ai_vid_chatbots_caption), play_button: 'center', letterbox: 'false', download_path: "//videos.code.org/ai_series/ai-chatbots.mp4" %>
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/views/ai_vid_chatbots.erb]: Update cobranded `How Chatbots and Large Language Models Work` video here, and new AWS download link

File: pegasus/sites.v3/code.org/views/ai_vid_creativity.erb [hunk 10]
```diff
@@ -1,3 +1,4 @@
 <div class="col-50" style="float: left; padding: 10px; margin-bottom: 40px">
+  <!-- Update cobranded video here, and new AWS download link -->
   <%=view :display_video_thumbnail, id: "ai_creativity", video_code: "gD1bEC9q54k", caption: hoc_s(:ai_vid_creativity_caption), play_button: 'center', letterbox: 'false', download_path: "//videos.code.org/ai_series/ai-creativity.mp4" %>
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/views/ai_vid_creativity.erb]: Update `Does AI Have Creativity and Imagination?` cobranded video here, and new AWS download link

**Comment by kelbyhawn** [comment]
@dmcavoy I left comments where the videos need to be updated

File: pegasus/sites.v3/code.org/views/how_ai_works_lessons.haml [hunk 11]
```diff
@@ -1,5 +1,6 @@
 - lesson_index = 7
 - video_id = ["what_is_ml", "computer_vision", "neural_networks", "ai_chatbots", "ai_creativity", "bias_and_access", "automation_and_privacy"]
+-# Update cobranded videos here - T45kEEBzrT8 and gD1bEC9q54k
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/views/how_ai_works_lessons.haml]: Update cobranded videos here (this is using a loop)

**Comment by dmcavoy** [comment]
Tested locally. Merging to get stakeholder final review early

**Pull request merged**